paddle/fluid/pybind/reader_py.cc

```diff
@@ -20,46 +20,134 @@
 #include <utility>
 #include <vector>
 #include "Python.h"
+#include "boost/optional.hpp"
+#include "gflags/gflags.h"
 #include "paddle/fluid/framework/ddim.h"
 #include "paddle/fluid/framework/reader.h"
 #include "paddle/fluid/imperative/layer.h"
 #include "paddle/fluid/imperative/tracer.h"
 #include "paddle/fluid/operators/reader/buffered_reader.h"
+#include "paddle/fluid/operators/reader/lod_tensor_blocking_queue.h"
 #include "paddle/fluid/operators/reader/py_reader.h"
 #include "paddle/fluid/platform/place.h"
 #include "pybind11/stl.h"
 
+DEFINE_bool(reader_queue_speed_test_mode, false,
```

Comment:
If it stays false, will it affect training speed? Is this true for the previous test data?

Reply:
This flag has been already existed for a long time. I have just moved it from `pybind.cc` to `reader_py.cc`. It is only used to test reader speed (get data from data queue but do not remove it). It should not be used for actual training and testing.